index.js

```diff
@@ -76,6 +76,9 @@ smartcar.isExpired = function(expiration) {
  * @param {Number} [paging.limit] - number of vehicles to return
  * @param {Number} [paging.offset] - index to start vehicle list
  * @return {Promise.<module:smartcar~VehicleIds>} A promise with the vehicle ids.
+ * @throws {SmartcarError} - an instance of SmartcarError.
```

Comment:
I notice this isn't showing up correctly on GitHub when you view the rendered version. `Throws` is on the same line as the return statement. Can you investigate?

Reply:
mm that's just how our tool renders it. feel free to add another task to investigate. I just ran `npm run docs`. most docs are on the same line unfortunately :(

Reply:
Roger that